flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/RowDataFileScanTaskReader.java

```diff
@@ -55,11 +56,18 @@
   private final Schema projectedSchema;
   private final String nameMapping;
   private final boolean caseSensitive;
+  private int[][] projectedFields;
```

Comment:
We have already had a `projectedSchema`,  why do we still need a `projectedFields` here ?  Shouldn't we just convert the `projectedFields` into a `projectedSchema`, and then use the projected schema to go though the whole read path ?